src/tokens/ERC5095.sol

```diff
@@ -279,8 +495,7 @@ contract ERC5095 is ERC20Permit, IERC5095 {
                 Safe.transfer(IERC20(underlying), r, returned);
```

Comment:
For some reason it wont let me comment on it, but line 491 should have the same patterns using buyUnderlying, but utilize the directly calculates shares and a values